lib/pengine/unpack.c

```diff
@@ -379,9 +379,13 @@ unpack_config(xmlNode * config, pe_working_set_t * data_set)
               pcmk__btoa(pcmk_is_set(data_set->flags,
                                      pcmk_sched_in_maintenance)));
 
-    set_config_flag(data_set, "start-failure-is-fatal", pe_flag_start_failure_fatal);
-    crm_trace("Start failures are %s",
-              pcmk_is_set(data_set->flags, pe_flag_start_failure_fatal)? "always fatal" : "handled by failcount");
+    set_config_flag(data_set, "start-failure-is-fatal",
+                    pcmk_sched_start_failure_fatal);
+    if (pcmk_is_set(data_set->flags, pcmk_sched_start_failure_fatal)) {
```

Comment:
forget it if I didn't get the real reason ...

Reply:
I just got sloppy for fencing_enabled :)